docs/wiki/[User-Guide]-Starter-Module-Terraform-Microsoft-Cloud-For-Sovereignty.md

```diff
@@ -0,0 +1,96 @@
+<!-- markdownlint-disable first-line-h1 -->
+The `microsoft_cloud_for_sovereignty` starter module provides full customization of the Sovereign Landing Zone using the `inputs.yaml` file. The `inputs.yaml` file provides the ability to enable and disable modules, configure module inputs and outputs, and configure module resources.
+A custom `inputs.yaml` file can be passed to the `inputConfigFilePath` argument of the ALZ PowerShell Module. This allows you to firstly design your Azure Landing Zone, and then deploy it.
+
+The default `inputs.yaml` file will need to be modified based off the documentation found [here][parameter_description_powershell_inputs_local_terraform_slz].
+
+Default input files can be found here:
+
+- [inputs-azure-devops-terraform-microsoft-cloud-for-sovereignty.yaml][example_powershell_inputs_azure_devops_terraform_microsoft_cloud_for_sovereignty]
+- [inputs-github-terraform-microsoft-cloud-for-sovereignty.yaml][example_powershell_inputs_github_terraform_microsoft_cloud_for_sovereignty]
+- [inputs-local-terraform-microsoft-cloud-for-sovereignty.yaml][example_powershell_inputs_local_terraform_microsoft_cloud_for_sovereignty]
+
+The following table describes the inputs required for the `microsoft_cloud_for_sovereignty` starter module.
+
+| Input | Placeholder | Description |
+| - | -- | --- |
+| `default_prefix` | `<prefix>` | Prefix value to append to all resources. |
+| `default_location` | `<location>` | This is the Azure region to deploy all SLZ resources into. |
+| `allowed_locations` | `<list of locations>` | This is a list of Azure regions all workloads running outside of the Confidential Management Group scopes are allowed to be deployed into. |
+| `allowed_locations_for_confidential_computing` | `<list of locations>` | This is a list of Azure regions all workloads running inside of the Confidential Management Group scopes are allowed to be deployed into. |
+| `iac` | `terraform` | For public preview, only `"terraform"` is supported. |
+| `bootstrap` | `alz_local` | For public preview, only `"alz_local"` is supported. |
+| `bootstrap_location` | `<location>` | Use the same value as the `default_location`. As of the current release, it is required but not used. |
+| `create_bootstrap_resources_in_azure` | `false` | For public preview, only `false` is supported. |
```

Comment:
Please do not include bootstrap inputs in this doc. They are documented here: https://github.com/Azure/ALZ-PowerShell-Module/wiki/%5BUser-Guide%5D-Quick-Start-Phase-2-Azure-DevOps